src/components/projects/projectId/data-browser/DataBrowser.tsx

```diff
@@ -169,7 +174,9 @@ export default function DataBrowser() {
     return (<>
         {projectId && <div className="flex flex-row h-full">
             <DataBrowserSidebar />
-            <DataBrowserRecords refetchNextRecords={getNextRecords} />
+            <DataBrowserRecords refetchNextRecords={getNextRecords} clearSearchRequest={() => {
+                setSearchRequest(SEARCH_REQUEST);
+            }} />
```

Comment:
inline function linked to complex subcomponent. best with useCallback to prevent expensive rerender